STM32F405RGTx/Core/Inc/svpwm_thread.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "VHZ.h"
+#include "main.h"
+#include "util.hpp"
+
+#ifndef INC_SVPWM_H_
+#define INC_SVPWM_H_
+
+class SVPWM{
+    public:
+        static float  ModIndex;
+		static float  Freq;

```

Comment:
can u rename `Freq` to something like `FundamentalFreq`?

Reply:
done